### ISSUE TITLE
Rename scoring section

### ISSUE THREAD

**Issue by jabdoa2** [missionpinball/mpf#901]
* It now also supports machine_vars. Maybe we can also come up with another name? (i guess Brian suggested that before also)
* We should also rename the inner "score:" section in the explicit syntax to something more explicit like "int" or "value" (we also support "float" and "string"). This looks counter-intuitively : 

```
scoring:
    my_event:
      score:
          score: 1000   # this keyword is confusing
          block: true
```

Users get that wrong. See: https://groups.google.com/d/msg/mpf-users/aLymXS77qCY/uiWippx0AAAJ for example

**Comment by qcapen** [comment]
Scoring is really a variable player, so what about renaming it to 'variable_player' for consistency?  Also, I like renaming the 'score:' parameter to 'int:' to be consistent with 'string:' and 'float:'.

**Comment by qcapen** [comment]
Fixed in PR #1146
